tensorflow/contrib/timeseries/python/timeseries/estimators.py

```diff
@@ -22,6 +22,7 @@
 from tensorflow.contrib.timeseries.python.timeseries import feature_keys
 from tensorflow.contrib.timeseries.python.timeseries import math_utils
 from tensorflow.contrib.timeseries.python.timeseries import model_utils
+from tensorflow.contrib.timeseries.python.timeseries import ts_head_lib
```

Comment:
Should this be "from tensorflow.contrib.timeseries.python.timeseries import head as ts_head_lib"?